tests/pip_install/whl_library/generate_build_bazel_tests.bzl

```diff
@@ -44,7 +120,13 @@ filegroup(
         "@pypi_foo//:whl",
     ] + select(
         {
-            "@platforms//os:windows": ["@pypi_colorama//:whl"],
+            "@//python/config_settings:is_python_3.9": ["@pypi_py39_dep//:whl"],
+            "@platforms//cpu:aarch64": ["@pypi_arm_dep//:whl"],
+            "@platforms//os:windows": ["@pypi_win_dep//:whl"],
+            ":is_cp310_linux_ppc": ["@pypi_py310_linux_ppc_dep//:whl"],
+            ":is_cp39_anyos_aarch64": ["@pypi_py39_arm_dep//:whl"],
+            ":is_cp39_linux_anyarch": ["@pypi_py39_linux_dep//:whl"],
+            ":is_linux_x86_64": ["@pypi_linux_intel_dep//:whl"],
```

Comment:
This doesn't look like a correct set of conditions?

The first 3 could all be true simultaneously and aren't specializations of each other, which would be an error.

Reply:
resolved on chat. This is just an artifact of the unittest manually constructing values to verify the select generation. the exlusivity logic is part of wheel.py